deploy/olm-catalog/qdr-operator/0.1.0/catalog-source.yaml

```diff
@@ -50,48 +51,84 @@ items:
                   version: v1
                 specDescriptors:
                 - description: The role and placement plan for the interconnect deployment
-                  displayName: DeploymentPlan
-                  path: deploymentPlan
+                  displayName: Deployment Plan Image
+                  path: deploymentPlan.image
+                  x-descriptors:
+                    - "urn:alm:descriptor:com.tectonic.ui:label"
+                - description: The role and placement plan for the interconnect deployment
+                  displayName: Deployment Placement
+                  path: deploymentPlan.placement
+                  x-descriptors:
+                    - "urn:alm:descriptor:com.tectonic.ui:label"
+                - description: The role and placement plan for the interconnect deployment
+                  displayName: Deployment Role
+                  path: deploymentPlan.role
+                  x-descriptors:
+                    - "urn:alm:descriptor:com.tectonic.ui:label"
+                - description: Listeners for incoming connections to the router
+                  displayName: Listener Port
+                  path: listeners.port
+                  x-descriptors:
+                    - "urn:alm:descriptor:com.tectonic.ui:label"
                 - description: Listeners for incoming connections to the router
-                  displayName: Listeners
-                  path: listeners
+                  displayName: Listener http
+                  path: listeners.http
+                  x-descriptors:
+                    - "urn:alm:descriptor:com.tectonic.ui:label"
```

Comment:
Can you explain what these changes do? There doesn't seem to be much additional information in them as to e.g. the type or valid values or description.

Reply:
This is to tell we want to display the value as text/label.
It is here just to be more specific, as we actually just want to display, instead of using any default x-descriptor that may have.

Reply:
Can you describe how it renders differently?

Reply:
with label x-descriptors: it shows '--'; otherwise, shows 'None'